crates/forge/src/cmd/bind.rs

```diff
@@ -112,31 +113,58 @@ pub struct BindArgs {
 }
 
 impl BindArgs {
-    pub fn run(self) -> Result<()> {
+    pub fn run(mut self) -> Result<()> {
         if self.ethers {
             eyre::bail!("`--ethers` bindings have been removed. Use `--alloy` (default) instead.");
         }
-
         if !self.skip_build {
```

Comment:
how it will work if I set skip build?

Reply:
what do you mean as not standard flag? Is it possible to set it from command line?

Reply:
as in normally you wouldn't pass it unless you know why are you doing it. by default it will recompile every time.